src/components/modals/ant-management/AddUndernameModal/AddUndernameModal.tsx

```diff
@@ -241,8 +250,12 @@ function AddUndernameModal({
                     customPattern={ARNS_TX_ID_ENTRY_REGEX}
                     validationPredicates={{
                       [VALIDATION_INPUT_TYPES.ARWEAVE_ID]: {
-                        fn: (id: string) =>
-                          arweaveDataProvider.validateArweaveId(id),
+                        fn: async (id: string) => {
```

Comment:
Is the `async` intended here?

Reply:
Yes, its a promise based validation system.